manage_breast_screening/notifications/tests/conftest.py

```diff
@@ -17,8 +17,13 @@ def commands_module_str(base_module_str) -> str:
     return f"{base_module_str}.management.commands"
 
 
-@pytest.fixture
-def mock_insights_logger(monkeypatch):
+@pytest.fixture(autouse=True)
```

Comment:
My bad, but out of interest why do we autouse this? I added a conftest to share the fixture but I think this might now mean the fixture is autoused by all tests?

Reply:
Ah sorry the commit message explains it very well.